MessageSigning/MessageSigning.md

```diff
@@ -0,0 +1,433 @@
+---
+CIP: TBD
```

Comment:
I'd be happy to assign this one a number already so we can reference it in the upcoming CIP about multisig wallets.